test/functional/pdo_sqlsrv/pdo_test_non_LOB_types.phpt

```diff
@@ -0,0 +1,78 @@
+--TEST--
+Test reading non LOB types
+--DESCRIPTION--
+A simpler version of sqlsrv test "test_sqlsrv_phptype_stream.phpt" for reading from 
+pre-populated tables [test_streamable_types] and [test_types]
+--SKIPIF--
+<?php require('skipif_azure_dw.inc'); ?>
+--FILE--
+<?php
+
+require('MsSetup.inc');
+
+function verifyResult($result)
+{
+    if (empty($result) || !is_array($result)) {
+        echo "Result is empty or not an array!\n";
+        return;
+    }
+    
+    $trimmedLen = 200;
+    $fullLen = 256;
+    $input = str_repeat('A', $trimmedLen);
+    for ($i = 0; $i < count($result); $i++) {
+        $expectedLen = ($i % 2 == 0) ? $fullLen : $trimmedLen;
+        $len = strlen($result[$i]);
+        if ($len != $expectedLen) {
+            echo "String length $len for column ". $i + 1 . " is unexpected!\n";
+        }
+        
+        $data = rtrim($result[$i]);
+        if ($data !== $input) {
+            echo "Result for column ". $i + 1 . " is unexpected:";
+            var_dump($result[$i]);
+        }
+    }
+}
+
+try {
+    $conn = new PDO("sqlsrv:server=$server; Database = $databaseName", $uid, $pwd);
+    $conn->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
+
+    // test the allowed non LOB column types
+    $tsql = "SELECT [char_short_type], [varchar_short_type], [nchar_short_type], [nvarchar_short_type], [binary_short_type], [varbinary_short_type] FROM [test_streamable_types]";
+    $stmt = $conn->query($tsql);
+
+    $result = $stmt->fetch(PDO::FETCH_NUM);
+    verifyResult($result);
+
+    // test not streamable types
+    $tsql = "SELECT * FROM [test_types]";
+    $stmt = $conn->query($tsql);
+    $result = $stmt->fetch(PDO::FETCH_NUM);
+    print_r($result);

```

Comment:
I'm unclear on what the purpose of this part is? I know the corresponding sqlsrv test tests fetching non-streamable types as streams, but since we can't stream data in pdo_sqlsrv is there any reason to have this part?

Reply:
Mainly to create a similar pdo test for using both as tests used in stress testing

Reply:
ok